app/views/products/index.html.erb

```diff
@@ -1,5 +1,15 @@
+<% if @user %>
+  <h1> Shop All <%= @user.merchant_name %>'s Products </h1>
```

Comment:
We might want to add another conditional here to display @user.username if @user.merchant_name does not exist, since it's not a required field.